src/main/java/world/bentobox/bentobox/BentoBox.java

```diff
@@ -594,4 +597,16 @@ public void reloadConfig() {
     public boolean isShutdown() {
         return shutdown;
     }
+
+    public List<String> getCommands(World world) {
```

Comment:
The only place this method is called is in src/main/java/world/bentobox/bentobox/listeners/flags/clicklisteners/CommandRankClickListener.java so that's where it should stay.